_sass/_front_page.scss

```diff
@@ -189,14 +189,15 @@ ul.banner-social-buttons {
   & img {
     margin: 25px;
   }
+
+  &.backers object {
+    width: 400px;
+    height: 280px;
```

Comment:
Can we keep the height auto instead of a fixed px? Just in case the number of backers increase in future, I feel this height might cause issues.

Reply:
Was thinking about this as well, but auto doesn’t seem to work, because it’s an embedded SVG and we don’t know the height of it.

The reason I added this is because the current widget is broken on mobile like so (and that CSS fixes it):
![image](https://user-images.githubusercontent.com/925062/103823559-9ae31800-5072-11eb-859c-3b4816452880.png)

Reply:
I see. Seems like there's not many easy solutions to this (given the embedded SVG), so approving and merging for now.